src/components/index.ts

```diff
@@ -116,6 +122,11 @@ const ComponentsPlugin: PluginObject<any> = {
         Vue.use(RadioPlugin);
         Vue.use(RadioGroupPlugin);
         Vue.use(RadioStylePlugin);
+
+        if (options) {
```

Comment:
This if statement is useless, the options objet is always initialized.

Reply:
Forgot to remove the IF after adding options = options || {};  I'll also use a default parameter for options instead of options = options || {};